commands/economy/setmoney.js

```diff
@@ -0,0 +1,97 @@
+const Discord = require("discord.js")
+const { COLOR } = process.env;
+
+module.exports = {
+  name:'setmoney',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mhazm/Saino-sama/pullRequest?prid=7537166)
```suggestion
  name:"setmoney",
```